access/validator/validator.go

```diff
@@ -429,6 +429,20 @@ func (v *TransactionValidator) checkSignatureDuplications(tx *flow.TransactionBo
 }
 
 func (v *TransactionValidator) checkSignatureFormat(tx *flow.TransactionBody) error {
+	for _, signature := range append(tx.PayloadSignatures) {
```

Comment:
why the append here? i thought if no new element is added, then the underlying array for the slice will still be the same, so it's not even providing a copy, therefore i'm not sure i understand the purpose of the append here?

Reply:
it's a mistake, I was appending two slices, then I removed one slice but didn't rermove the `append`. Good catch